src/ashell/shell-state.c

```diff
@@ -138,7 +138,7 @@ int32_t ashell_disk_usage(char *buf)
     ssize_t size = fs_size(file);
     fs_close(file);
 
-    printf("%5ld %s\n", size, filename);
+    printf("%d %s\n", size, filename);
```

Comment:
You dropped the 5 width specifier, and this won't build with 0.8.2 because of format issue. Also below.